### ISSUE TITLE
Unholy Death Knight: Summon Gargoyle

### ISSUE THREAD

**Issue by APTX4869s** [gamefreedomgit/Maelstrom#2019]
![Gargoyle](https://user-images.githubusercontent.com/57665529/214539433-7ce370ff-e63f-4aff-b6d4-9cdf79323327.png)
![Raise dead](https://user-images.githubusercontent.com/57665529/214539453-a643bb9c-9466-4763-be9e-cb225bac7616.png)

**Description:**
If you attack a dummy while having your pet ( raise dead ) assisting you, the Gargoyle won't cast anything and it just follows you around.
But if you attack a dummy without your pet ( raise dead ), the Gargoyle casts normally and does damage.

**How to reproduce:**
First attack the dummy then summon your Gargoyle without your ghoul ( raise dead ), you will see that the gargoyle casts "gargoyle strike" and works fine.
then attack the dummy and let your ghoul assists you ( already summoned ), after that summon gargoyle. You will see that the gargoyle doesn't cast " gargoyle strike " and just follows you around.

to sum it up, Gargoyle does nothing as long as the ghoul is summoned.
**How it should work:**
Both the gargoyle and the ghoul should assist you in combat.

https://www.youtube.com/watch?v=y0tfUCk7FbI
 at 2:07 you can see he's having both pets up and they do attack the dummy together.
